libs/features/personalization/personalization.js

```diff
@@ -427,12 +454,11 @@ export function handleCommands(commands, rootEl = document, forceInline = false)
       registerInBlockActions(cmd, manifestId, targetManifestId);
       return;
     }
-
+    const el = getSelectedElement({ selector, rootEl });
     if (action in COMMANDS) {
-      const el = getSelectedElement(selector, action, rootEl);
-      COMMANDS[action](el, target, manifestId, targetManifestId);
+      // eslint-disable-next-line max-len
```

Comment:
instead of disabling max length, I believe the preferred method is to put the elements of the object you are passing on their own lines.